core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.html

```diff
@@ -0,0 +1,11 @@
+<h2 mat-dialog-title>Visualization</h2>
+
+<mat-dialog-content>
+  <div id="texera-result-chart-content">
```

Comment:
also add a comment here that references to `VisualizationPanelContentComponent.CHART_ID`, so both places point to each other.